onboarding/asset-management-directions.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: Directions for information management aspects of the project.
```

Comment:
maybe a bit shorter?

`Information Management` or `Guide for Information Management`